pkgs/racket-doc/scribblings/getting-started/getting-started.scrbl

```diff
@@ -11,8 +11,11 @@ executable will run a command-line Read-Eval-Print-Loop
 (@tech[#:doc '(lib "scribblings/guide/guide.scrbl")]{REPL}).
 
 On Windows, you can start DrRacket from the @onscreen{Racket} entry in the
-Start menu. In Windows Vista or newer, you can just type @exec{DrRacket}.  You can
-also run it from its folder, which you can find in @onscreen{Program Files} →
+Start menu. In Windows Vista or newer, you can just type @exec{DrRacket}.
+@margin-note*{If you prefer, you can also work with the command-line @exec{racket} 
+interpreter (see @hyperlink["https://docs.racket-lang.org/guide/racket.html"]{Running racket and gracket}) 
```

Comment:
For links within Racket Documentation itself, please use `secref` or `seclink`.

Reply:
Also, the "Running racket and gracket" section kinda assumes that you have Linux installed. I don't know if it will be confusing to make this link.

Reply:
> the "Running racket and gracket" section kinda assumes that you have Linux installed. I don't know if it will be confusing to make this link.

this is an exact copy of the text at https://docs.racket-lang.org/guide/intro.html
<img width="719" alt="image" src="https://github.com/racket/racket/assets/108047/1200556d-10cf-43a3-b683-0c1b5474b10e">

Reply:
> For links within Racket Documentation itself, please use secref or seclink.

changed to secref links

Reply:
> the "Running racket and gracket" section kinda assumes that you have Linux installed. I don't know if it will be confusing to make this link.

removed link and supporting text